app/server/tests/db/sessionStore.test.ts

```diff
@@ -0,0 +1,35 @@
+import { SessionStore } from "../../src/db/sessionStore";
+
+// Mock Date.now to return hardcoded date
+Date.now = jest.spyOn(Date, "now").mockImplementation(() => new Date(2022, 0, 24, 17).getTime()) as any;
```

Comment:
what is this? base 0 month but base 1 year and day?!

Reply:
OMG it is, that's horrific https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

Reply:
🤣 oh yes